installer/ansible/script/virtualenv3_exec.j2

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env bash
```

Comment:
this file may not required. Please check

Reply:
deleted this file